index.html

```diff
@@ -886,6 +886,152 @@ <h5>Timing Properties</h5>
             </p>
           </section>
         </section>
+
+        <section>
+          <h4>Deferred resolution of timecode-based synchronisation</h4>
+          <p>In workflows that create <a>DAPT documents</a> by conversion from formats
+            that synchronise content using timecode to match time stamps
+            in the <a>related media object</a>,
+            a common issue occurs if the timecode of
+            the beginning of the media is not known at the time of conversion.
+          </p>
+          <p>The optional <a>Media Zero Timecode</a> and
+            <a>Start of Programme Timecode</a> properties can be used to
+            identify when there is a possible synchronisation error,
+            and to resynchronise the document when all
+            the required information is known.</p>
+          <p>These properties are provided as metadata only and
+            are not intended to be used to perform direct synchronisation offsets
+            during presentation.
+          </p>
```

Comment:
I'm still trying to understand the feature here and how it differs from the existing EBU-TT feature.

Is the proposed text trying to say the following:
> As defined in TTML2, each DAPT document has a timeline on which Script Events are positioned using their begin attributes. For synchronization purposes it is useful to map the DAPT document timeline with the related media object timeline.
In some cases, the mapping is simple, e.g. media time 0 in the related media object corresponds to the origin of the DAPT document timeline. In other cases, a timecode may be necessary. DAPT provides the ability to provide 2 different timecodes: 
> * The "Start of the Programme Timecode" is the timecode of the time instant in the related media object corresponding to the origin of the DAPT document timeline. It is represented in a DAPT document by a documentStartOfProgramme element, defined in EBUT-TT, using the EBU-TT namespace, located in a metadata element in the head. 
>
> (and either one of the alternatives below)
> * The "Media Zero Timecode" is the timecode of the first media data in the related media object. It is represented in DAPT documents by the mediaZeroTimecode element, in the DAPT namespace, located in a metadata element in the head.
> * The "First Script Event Timecode" is the timecode of the time instant in the related media object corresponding to the first script event in the DAPT document. It is represented in DAPT documents by the XXX element, in the DAPT namespace, located in a metadata element in the head.
>
> NOTE: Depending on the script and related media object workflows, there may not be a timecode in the related media object corresponding to the origin of the DAPT document timeline, for example if the related media object has been edited to remove some data at its start. In that case, either the "Start of the Programme Timecode" and all the Script Event begin times have to be edited, or alternatively the "Start of the Programme Timecode" cannot be used but the "First Script Event Timecode" or "Media Zero Timecode" can be used.

Reply:
No, very much not that!

Start of programme timecode is the timecode in the _related media_ corresponding to the beginning of the related media object's programme content.

Media Zero timecode is a timecode that media time zero in the _DAPT document_ maps to.

If the DAPT document is not synchronised to the related media object's programme content, those two values will differ.

If the start of programme timecode is unknown, then you don't know if they're in sync or not.

If both are present and the same, then you have some reassurance that the DAPT document is synchronised with the media.

Reply:
>the timecode of the time instant in the related media object corresponding to the origin of the DAPT document timeline

The issue is that documents that are not synchronised to the related media object do not satisfy the "corresponding to the origin of the DAPT document timeline" part of this sentence.